diff --git a/js/views_slideshow.js b/js/views_slideshow.js
index b063eed..3070fb1 100644
--- a/js/views_slideshow.js
+++ b/js/views_slideshow.js
@@ -108,7 +108,7 @@
    */
   Drupal.viewsSlideshowControlsText.pause = function (options) {
     var pauseText = Drupal.theme.prototype['viewsSlideshowControlsPause'] ? Drupal.theme('viewsSlideshowControlsPause') : '';
-    $('#views_slideshow_controls_text_pause_' + options.slideshowID + ' a').text(pauseText);
+    $('#views_slideshow_controls_text_pause_' + options.slideshowID + ' a').html(pauseText);
     $('#views_slideshow_controls_text_pause_' + options.slideshowID).removeClass('views-slideshow-controls-text-status-play');
     $('#views_slideshow_controls_text_pause_' + options.slideshowID).addClass('views-slideshow-controls-text-status-pause');
   };
@@ -118,19 +118,19 @@
    */
   Drupal.viewsSlideshowControlsText.play = function (options) {
     var playText = Drupal.theme.prototype['viewsSlideshowControlsPlay'] ? Drupal.theme('viewsSlideshowControlsPlay') : '';
-    $('#views_slideshow_controls_text_pause_' + options.slideshowID + ' a').text(playText);
+    $('#views_slideshow_controls_text_pause_' + options.slideshowID + ' a').html(playText);
     $('#views_slideshow_controls_text_pause_' + options.slideshowID).removeClass('views-slideshow-controls-text-status-pause');
     $('#views_slideshow_controls_text_pause_' + options.slideshowID).addClass('views-slideshow-controls-text-status-play');
   };
 
   // Theme the resume control.
   Drupal.theme.prototype.viewsSlideshowControlsPause = function () {
-    return Drupal.t('Resume');
+    return '<img src="/sites/all/themes/ninesixty/images/play.png" alt="Resume" />';
   };
 
   // Theme the pause control.
   Drupal.theme.prototype.viewsSlideshowControlsPlay = function () {
-    return Drupal.t('Pause');
+    return '<img src="/sites/all/themes/ninesixty/images/pause.png" alt="Pause" />';
   };
 
   /**
